Tests/CatalogItems/Get-RsFolderContent.Tests.ps1

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+Describe "Get-RsFolderContent" {
```

Comment:
@erimaja, could you delete the item you create in the tests using Remove-RsCatalogItem in the It function?